diff -Naur shared-mime-info-2.1.orig/data/meson.build shared-mime-info-2.1/data/meson.build
--- shared-mime-info-2.1.orig/data/meson.build	2021-01-19 10:15:58.618356556 +0000
+++ shared-mime-info-2.1/data/meson.build	2021-01-19 10:17:06.897894749 +0000
@@ -21,15 +21,3 @@
   [ 'its/shared-mime-info.loc', 'its/shared-mime-info.its', ],
   install_dir : get_option('datadir') / 'gettext/its'
 )
-
-custom_target('shared-mime-info-spec-html',
-    input : 'shared-mime-info-spec.xml',
-    output: 'shared-mime-info-spec-html',
-    command: [
-        xmlto,
-        '-o', '@OUTPUT@',
-        'html-nochunks',
-        '@INPUT@',
-    ],
-    build_by_default: true,
-)
diff -Naur shared-mime-info-2.1.orig/meson.build shared-mime-info-2.1/meson.build
--- shared-mime-info-2.1.orig/meson.build	2021-01-19 10:15:58.618356556 +0000
+++ shared-mime-info-2.1/meson.build	2021-01-19 10:16:36.042893679 +0000
@@ -20,9 +20,7 @@
 ###############################################################################
 # Find tools
 
-itstool = find_program('itstool')
-xmllint = find_program('xmllint')
-xmlto   = find_program('xmlto')
+# Nobody needs that.. c'mon upstream!
 
 ###############################################################################
 # Find xdgmime
diff -Naur shared-mime-info-2.1.orig/tests/meson.build shared-mime-info-2.1/tests/meson.build
--- shared-mime-info-2.1.orig/tests/meson.build	2021-01-19 10:15:58.625362460 +0000
+++ shared-mime-info-2.1/tests/meson.build	2021-01-19 10:18:15.237483548 +0000
@@ -47,11 +47,6 @@
     args: freedesktop_org_xml,
 )
 
-test('xmllint freedesktop.org.xml',
-    xmllint,
-    args: [ '--noout', '--valid', freedesktop_org_xml, ],
-)
-
 test('Case sensitivity',
     find_program('test-case-sensitivity.sh'),
 )
@@ -63,8 +58,3 @@
 
 its20_elements_rng = meson.source_root() / 'data/its/its20-elements.rng'
 shared_mime_info_its = meson.source_root() / 'data/its/shared-mime-info.its'
-
-test('ITS validation',
-     xmllint,
-     args: [ '--noout', '--relaxng', its20_elements_rng, shared_mime_info_its, ],
-)
