diff -up Vulkan-ValidationLayers-sdk-1.2.182.0/layers/vk_layer_config.cpp.dma Vulkan-ValidationLayers-sdk-1.2.182.0/layers/vk_layer_config.cpp
--- Vulkan-ValidationLayers-sdk-1.2.182.0/layers/vk_layer_config.cpp.dma	2021-08-07 08:13:46.344320103 +1000
+++ Vulkan-ValidationLayers-sdk-1.2.182.0/layers/vk_layer_config.cpp	2021-08-07 08:14:22.130893289 +1000
@@ -68,7 +68,7 @@ class ConfigFile {
 
 static ConfigFile layer_config;
 
-string GetEnvironment(const char *variable) {
+VK_LAYER_EXPORT string GetEnvironment(const char *variable) {
 #if !defined(__ANDROID__) && !defined(_WIN32)
     const char *output = getenv(variable);
     return output == NULL ? "" : output;
diff -up Vulkan-ValidationLayers-sdk-1.2.182.0/layers/vk_layer_config.h.dma Vulkan-ValidationLayers-sdk-1.2.182.0/layers/vk_layer_config.h
--- Vulkan-ValidationLayers-sdk-1.2.182.0/layers/vk_layer_config.h.dma	2021-08-07 08:14:04.352608540 +1000
+++ Vulkan-ValidationLayers-sdk-1.2.182.0/layers/vk_layer_config.h	2021-08-07 08:14:15.237782885 +1000
@@ -33,7 +33,7 @@
 #define SECONDARY_VK_REGISTRY_HIVE_STR "HKEY_CURRENT_USER"
 #endif
 
-std::string GetEnvironment(const char *variable);
+VK_LAYER_EXPORT std::string GetEnvironment(const char *variable);
 
 #ifdef __cplusplus
 extern "C" {
