diff -purN xfce4-macmenu-plugin.orig//macmenu-applet.c xfce4-macmenu-plugin/macmenu-applet.c
--- xfce4-macmenu-plugin.orig//macmenu-applet.c	2010-12-17 09:16:12.000000000 +0100
+++ xfce4-macmenu-plugin/macmenu-applet.c	2010-12-17 09:17:24.000000000 +0100
@@ -15,6 +15,7 @@
 
 #ifdef FOR_XFCE
 // begin FOR_XFCE
+#include <libxfce4util/libxfce4util.h>
 #include <libxfce4panel/xfce-panel-convenience.h>
 #include <libxfce4panel/xfce-panel-plugin.h>
 typedef XfcePanelPlugin AppletType;
diff -purN xfce4-macmenu-plugin.orig//xfce4-macmenu-plugin.desktop xfce4-macmenu-plugin/xfce4-macmenu-plugin.desktop
--- xfce4-macmenu-plugin.orig//xfce4-macmenu-plugin.desktop	2010-12-17 09:16:12.000000000 +0100
+++ xfce4-macmenu-plugin/xfce4-macmenu-plugin.desktop	2010-12-17 09:21:56.000000000 +0100
@@ -4,4 +4,4 @@ Encoding=UTF-8
 Name=Mac Menu
 Comment=Mac Menubar applet for gtk2-aqd
 Icon=gnome-fs-home
-X-XFCE-Exec=/opt/xfce4/libexec/xfce4/panel-plugins/xfce4-macmenu-plugin
+X-XFCE-Exec=/usr/lib/xfce4/panel-plugins/xfce4-macmenu-plugin
