diff -purN libsexy-0.1.11.orig/configure libsexy-0.1.11/configure
--- libsexy-0.1.11.orig/configure	2011-12-12 09:29:50.000000000 +0100
+++ libsexy-0.1.11/configure	2011-12-12 09:31:21.000000000 +0100
@@ -20268,6 +20268,7 @@ echo "${ECHO_T}yes" >&6; }
 fi
 
 
+PACKAGE_DEPS=$pkg_modules
 
 GLIB_GENMARSHAL=`pkg-config --variable=glib_genmarshal glib-2.0`
 
diff -purN libsexy-0.1.11.orig/configure.ac libsexy-0.1.11/configure.ac
--- libsexy-0.1.11.orig/configure.ac	2011-12-12 09:29:50.000000000 +0100
+++ libsexy-0.1.11/configure.ac	2011-12-12 09:32:11.000000000 +0100
@@ -86,6 +86,7 @@ pkg_modules="pango >= 1.4.0, glib-2.0 >=
 PKG_CHECK_MODULES(PACKAGE, [$pkg_modules])
 AC_SUBST(PACKAGE_CFLAGS)
 AC_SUBST(PACKAGE_LIBS)
+AC_SUBST(PACKAGE_DEPS, $pkg_modules)
 
 GLIB_GENMARSHAL=`pkg-config --variable=glib_genmarshal glib-2.0`
 AC_SUBST(GLIB_GENMARSHAL)
diff -purN libsexy-0.1.11.orig/libsexy.pc.in libsexy-0.1.11/libsexy.pc.in
--- libsexy-0.1.11.orig/libsexy.pc.in	2011-12-12 09:29:50.000000000 +0100
+++ libsexy-0.1.11/libsexy.pc.in	2011-12-12 09:32:54.000000000 +0100
@@ -6,7 +6,8 @@ includedir=${prefix}/include
 Name: libsexy
 Description: Doing naughty things to good widgets
 Version: @VERSION@
-Libs: -L${libdir} -lsexy @PACKAGE_LIBS@
-Cflags: -I${includedir} @PACKAGE_CFLAGS@
+Requires.private: @PACKAGE_DEPS@
+Libs: -L${libdir} -lsexy
+Cflags: -I${includedir}
 
 
