diff -pruN libtorrent-rasterbar-1.2.11.orig/configure.ac libtorrent-rasterbar-1.2.11/configure.ac
--- libtorrent-rasterbar-1.2.11.orig/configure.ac	2020-12-28 16:23:27.250050285 +1100
+++ libtorrent-rasterbar-1.2.11/configure.ac	2020-12-28 16:24:25.168717701 +1100
@@ -328,8 +328,8 @@ AS_CASE(["$ARG_ENABLE_DEBUG"],
   ["no"], [
       AC_MSG_RESULT([no])
       AC_DEFINE([NDEBUG],[1],[Define to disable debug code.])
-      #COMPILETIME_OPTIONS="$COMPILETIME_OPTIONS -DNDEBUG "
-      DEBUGFLAGS="-g0 -Os"
+      COMPILETIME_OPTIONS="$COMPILETIME_OPTIONS -DNDEBUG "
+      #DEBUGFLAGS="-g0 -Os"
    ],
   [AC_MSG_RESULT([$ARG_ENABLE_DEBUG])
    AC_MSG_ERROR([Unknown option "$ARG_ENABLE_DEBUG". Use either "yes" or "no".])]
